src/main/scala/com/gu/newsletterlistcleanse/NewsletterConfig.scala

```diff
@@ -28,7 +30,9 @@ object NewsletterConfig {
       cutOffSqsUrl = config.getString("cutOffSqsUrl"),
       cleanseListSqsUrl = config.getString("cleanseListSqsUrl"),
       archiveFilterSet = config.getStringList("archiveFilterList").asScala.toSet,
-      dryRun = if (config.hasPathOrNull("dryRun")) config.getBoolean("dryRun") else true
+      dryRun = if (config.hasPathOrNull("dryRun")) config.getBoolean("dryRun") else true,
+      backupBucketName = config.getString("backupBucketName"),
+      subscribeUsers = if (config.hasPathOrNull("subscribeUsers")) config.getBoolean("subscribeUsers") else false,
```

Comment:
Adding in the resubscribe flag to the config so that we can turn it on should we need to revert changes. It's got a default of `false`, but this seemed more sensible than trying to pass it into the handler and then have it work its way down through multiple functions.